test.lisp

```diff
@@ -1,22 +1,80 @@
 (in-package :cl)
 
 (defpackage :inferior-shell-test
-  (:use :cl :inferior-shell :hu.dwim.stefil))
+  (:use :cl :inferior-shell :fiveam))
 
 (in-package :inferior-shell-test)
 
 (declaim (optimize (speed 1) (debug 3) (space 3)))
 
-(defsuite* (test-suite
-            :in root-suite
-            :documentation "Testing inferior-shell"))
+(def-suite inferior-shell-test
+  :description "Suite of tests for inferior shell.")
 
-(defun do-test-inferior-shell ()
-  (is (equal (run/ss '(echo (1) "2" (+ 3))) "1 2 3"))
-  (is (equal (run/ss "echo 1    2        3") "1 2 3"))
-  (is (equal (run/ss `(pipe (echo (+ hel "lo,") world)
-                            (tr "hw" "HW") (sed -e "s/$/!/")))
-             "Hello, World!")))
+(def-test basics (:suite inferior-shell-test)
```

Comment:
Can you use `in-suite` or some such to avoid having to specify `(:suite ...)` in every test?